app/scripts/controllers/mmi-controller.js

```diff
@@ -161,11 +165,7 @@ export default class MMIController extends EventEmitter {
           }
         }
 
-        const txList = this.txController.txStateManager.getTransactions(
-          {},
-          [],
-          false,
-        ); // Includes all transactions, but we are looping through keyrings. Currently filtering is done in updateCustodianTransactions :-/
+        const txList = this.getTransactions(); // Includes all transactions, but we are looping through keyrings. Currently filtering is done in updateCustodianTransactions :-/
```

Comment:
As we're currently bound to the same state manager method, we have to provide the arguments since it defaults to filtering by current network which this old code was explicitly disabling.

Reply:
Sorry, added the arguments back.